lib/server.js

```diff
@@ -0,0 +1,11 @@
+import app from './services/app';
+
+const PORT = 3000;
+
+app.listen(PORT, (err) => {
+  if (err) {
+    console.log(err);
+  } else {
+    console.log(`Server listen on port: ${PORT}`);
```

Comment:
what about logger?

Reply:
I think, this pr is about project structure, but not about logger:)
As for me we need to move a feature about logger in separate pr. Step by step. @inst-event-emitter team, what do you think?

Reply:
ok, I'll create new issue